src/game/etj_timerun_v2.cpp

```diff
@@ -964,16 +926,14 @@ void ETJump::TimerunV2::checkRecord(Player *player) {
 
   _sc->postTask(
       [this, activeRunName, userId, completionTime, playerName, metadata,
-        checkpoints,
-        clientNum]() {
+       checkpoints, clientNum]() {
         /*
          * We want to check the record for all seasons.
          *
          * If player makes a new overall record, it is broadcasted
-         * If player makes a new relevant season record and there's no overall record,
-         * it is broadcasted
-         * If player makes a new season record, it is printed for the player but not
-         * for others
+         * If player makes a new relevant season record and there's no overall
```

Comment:
Formatter seems to be messing up the comment formatting a bit but it is what it is